README.md

```diff
@@ -25,7 +25,7 @@ If you want to run jaeger-logzio all-in-one (logz.io storage, Jaeger agent, coll
 | REGION | **Default**: `us` <br> Two-letters region code. Replace `us` with your region's code. For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html). |
 | CUSTOM_LISTENER_URL | Use this to set a custom listener URL (e.g http://localhost:9200). This config will override the region variable.|
 | CUSTOM_API | Use this to set an api URL (e.g http://localhost:9200/_msearch). This config will override the region variable.|
-
+| GRPC_STORAGE_PLUGIN_LOG_LEVEL | **Default**: `warn` <br> Log level to omit. Applies to the storage integration only. |
```

Comment:
@yyyogev Not sure I understand this one. Is it that the log level is ignored, or that any logs below this level are sent?